lib/utapi/utilities.js

```diff
@@ -114,10 +134,9 @@ export function listBucketMetrics() {
         }
         timeRange.push(numEnd);
     }
-
-    const bucketArr = buckets.split(',');
-    _listBucketMetrics(host, port, bucketArr, timeRange,
-        accessKey, secretKey, verbose, ssl);
+    const metricTypeArr = commander[metric].split(',');
```

Comment:
Can you call this `resources` instead of `metricTypeArr`? You can add a comment below saying resources is comma-separated list of buckets/accounts.